src/main/java/com/wolox/training/services/UserService.java

```diff
@@ -129,5 +133,6 @@ private void adaptUserDTOToUserModel(UserDTO userDTO, User user) {
         user.setUsername(userDTO.getUsername());
         user.setBirthdate(userDTO.getBirthdate());
         user.setName(userDTO.getName());
+        user.setPassword(passwordEncoder.encode(userDTO.getPassword()));
```

Comment:
The password should be set in a sepparate endpoint dedicated to it. 
Remove this line, add another endpoint for doing this and adapt as necessary.